packages/rich-text/src/special-characters.js

```diff
@@ -1,2 +1,5 @@
+/**
+ * Keycode for line separator.
```

Comment:
This is a string containing the "line separator" character, not a keycode?

Reply:
Ah, wasn't sure what it was :)

Reply:
Fixed at 70647bc

Reply:
Thanks! :)